extensions/security-jpa-common/runtime/src/main/java/io/quarkus/security/jpa/common/runtime/JpaIdentityProviderUtil.java

```diff
@@ -70,4 +73,13 @@ public static Password getMcfPassword(String pass) {
             throw new RuntimeException(e);
         }
     }
+
+    public static void passwordAction(PasswordType type) {
+        String uuid = UUID.randomUUID().toString();
+        if (type == PasswordType.CLEAR) {
+            ClearPassword.createRaw(ClearPassword.ALGORITHM_CLEAR, uuid.toCharArray());
+        } else {
```

Comment:
There is also a custom type but I think a BcryptUtil action is enough to cover it